src/neuroconv/tools/path_expansion.py

```diff
@@ -64,23 +64,31 @@ def expand_paths(self, source_data_spec: Dict[str, dict]) -> List[DeepDict]:
         ... )
         """
 
-        session_keys = {"session_start_time", "session_id", "subject_id"}
+        # session_keys = {"session_start_time", "session_id", "subject_id"}
```

Comment:
```suggestion

```
Do we need this?

Reply:
I don't think so? I got rid of that in the recent commit.